src/functions/events/v2.ts

```diff
@@ -9,7 +9,14 @@ export const STORAGE_EVENTS = [
 
 export const FIREBASE_ALERTS_PUBLISH_EVENT = "google.firebase.firebasealerts.alerts.v1.published";
 
+export const BEFORE_CREATE_EVENT = "providers/cloud.auth/eventTypes/user.beforeCreate";
+
+export const BEFORE_SIGN_IN_EVENT = "providers/cloud.auth/eventTypes/user.beforeSignIn";
+
+export const AUTH_BLOCKING_EVENTS = [BEFORE_CREATE_EVENT, BEFORE_SIGN_IN_EVENT] as const;
```

Comment:
IDK if I really love having this defined as a separate event type. I saw some places where only one type is handled and this also gets in the way of GCIP offering a new v2 api (not that they're super inclined to anyway).

In reality, events.v1 and events.v2 isn't about GCF versions as much as it is "eventflow" or "eventarc". It's OK that these event types will be handled by a v2 function and it's possible that we'll be forced to add support for all of events.v1 in v2 functions.

Reply:
Would you recommend just dropping v1 events entirely?